### ISSUE TITLE
multi-part PDF not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#420]
```
What steps will reproduce the problem?
1. Open a multi-part PDF
2. Open Outline -- outline links to other files do not appear.
3. Tap page-link to other file; nothing happens

What is the expected output?
- Outline should show links to other PDF files (usually in same folder)
- PageLinks and OutlineLinks to other PDF files should open the other PDF file 
and target the specified page.

What do you see instead?
- PageLinks don't work
- Outline is missing external links

What version of the product are you using?
1.6.4

On what operating system?
Android 2.3.3 - 4.1.2

What device are you using?
Various

Please provide any additional information below.
These are GOTOR links, and the filespec is available in muPDF. Just need to 
provide a way to switch documents (using intents?) internally. Extra credit if 
the navigation history maintains a path through multiple files.
```

Original issue reported on code.google.com by `ugra...@323go.com` on 7 Jan 2013 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
Multi-part pdf support will be added in future version along with 
multi-document feature.
```

Original comment by `Andrei.K...@gmail.com` on 7 Jan 2013 at 8:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 10 Jan 2013 at 5:12
- Changed state: **Postponed**
- Added labels: **Type-Enhancement**, **multi-document**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 10:22
- Changed state: **Fixed**
